system-monitor@paradoxxx.zero.gmail.com/metadata.json

```diff
@@ -1,5 +1,5 @@
 {
-    "shell-version": ["3.4","3.6","3.8","3.10", "3.12", "3.14", "3.16", "3.18", "3.20", "3.22", "3.24", "3.26", "3.28", "3.30"],
+    "shell-version": ["3.32"],
```

Comment:
Looks like ES6 is supported starting from gnome-shell 3.26. I've tested your changes in Ubuntu 18.04 / gnome-shell 3.28 and everything works. I guess this should be:
`"shell-version": ["3.26", "3.28", "3.30", "3.32"],`
instead?